docs/Installation.md

```diff
@@ -222,39 +236,36 @@ The user needs to define limits for the configured triggers. As highlighted in t
 
 The user also needs to define the lowest and highest values for the trigger. For example, for the temperature parameter, the range is set from 0 to 100 °C. The figure below shows the values provided for the temperature trigger, and the same process is followed for the liquid level parameter.
 
-![machine_monitor](graphics/Machine_Monitor_Rule_Two_Limits.png)
+![Secondrulelimits](graphics/Secondrulelimits.PNG)
 
 Similar to the first rule, the user can create instructions and add maintenance documents if needed.
 
 ### Configuring the third rule
 
-Click "New Maintenance Rule" to create a third rule "Quality Control - Defective Bottles" with the rule type "Intervals". This rule monitors the quality and efficiency of the production line based on the remaining parameter "numberFaulty". As highlighed in the figure below, once the number of defective bottles reach 200 bottle, the rule will be triggered and the service staff will have to carry out the linked instruction.
+Click "New Monitoring Rule" to create a third rule "Quality Control - Defective Bottles" with the rule type "Intervals". This rule monitors the quality and efficiency of the production line based on the remaining parameter "numberFaulty". As highlighed in the figure below, once the number of defective bottles reach 200 bottle, the rule will be triggered and the service staff will have to carry out the linked instruction.
 
-![machine_monitor](graphics/Machine_Monitor_Third_Rule_Limit.png)
+![Thirdrule](graphics/Thirdrule.PNG)
 
 The procedure to create this rule is the same as the two previous rules. If everything has been done correctly, the user can now see the available templates, as highlighted in the figure below:
 
-![machine_monitor](graphics/Machine_Monitor_Avaliable_Rules.png)
+![monitoringrulesoverview](graphics/monitoringrulesoverview.PNG)
 
 ### Assigning rules to assets
 
-The three rules are now configured and ready to be associated with the corresponding assets. To assign rules to assets, the user needs to open the "Overview" tab and click on "Asset Configuration" or "Go to Asset Configuration":
+The three rules are now configured and ready to be associated with the corresponding assets. To assign rules to assets, the user needs to open the "Overview" tab and click on "Add rule" and select "Assign predefined monitoring rules" as highlighted in the figure below:
 
-![machine_monitor](graphics/Machine_Monitor_Asset_Configuration.png)
+![Addruleasset](graphics/Addruleasset.PNG)
```

Comment:
In assigning rules to assists, the screen shots are taken from the parent asset " edge" and not from the child asset. This should be changed to avoid confusion

Reply:
I changed the child asset name